frontend/src/routes/Confirm/index.js

```diff
@@ -22,20 +23,20 @@ export default function Confirm(props) {
 
             <div className={classNames("btn-group", styles.buttons)}>
 
-                <Fab style={{ background: "#EA2027" }} size="large" className="fab" variant="extended" onClick={() => { handleClick('positive'); props.history.push("/questions") }}>
+                <Fab style={{ background: "#EA2027" }} size="large" className="fab" variant="extended" onClick={() => { handleClick(testStatus.POSITIVE); props.history.push("/questions") }}>
                     <span>YES, TESTED POSITIVE</span>
                 </Fab>
-                <Fab style={{ background: "#9206FF" }} size="large" className="fab" variant="extended" onClick={() => { handleClick('negative'); props.history.push("/questions") }}>
+                <Fab style={{ background: "#9206FF" }} size="large" className="fab" variant="extended" onClick={() => { handleClick(testStatus.NEGATIVE); props.history.push("/questions") }}>
                     <span>YES, TESTED NEGATIVE</span>
                 </Fab>
-                <Fab style={{ background: "#0559FD" }} size="large" className="fab" variant="extended" onClick={() => { handleClick('not tested'); props.history.push("/questions") }}>
+                <Fab style={{ background: "#0559FD" }} size="large" className="fab" variant="extended" onClick={() => { handleClick(testStatus.NOT_TESTED); props.history.push("/questions") }}>
                     <span>NO, I HAVE NOT</span>
                 </Fab>
 
             </div>
-            <Fab style={{ background: "#9206FF" }} size="medium" className="fab back-btn" onClick={() => props.history.push(isSick === "sick" ? '/alert' : '/onboard')}>
+            <Fab style={{ background: "#9206FF" }} size="medium" className="fab back-btn" onClick={() => props.history.push(sick === sicknessStatus.SICK ? '/alert' : '/onboard')}>
```

Comment:
Since we're tinkering here, all these URLs could be drawn from the `paths` constant available on the `routes` folder

Reply:
will change!